engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeMethodNode.java

```diff
@@ -621,6 +634,7 @@ Object doConvertDateTime(
     }
   }
 
+  @NonIdempotent
```

Comment:
What an [@NonIdempotent annotation](https://www.graalvm.org/truffle/javadoc/com/oracle/truffle/api/dsl/NonIdempotent.html)!

What is preventing this specialization to be idempotent?

Reply:
This annotation should not have been here. It seems like a leftover of my previous endeavors.